### DISCUSSION TITLE
gh-deploy creates user.github.io/repo/site

### DISCUSSION THREAD

**Discussion by flywire** [squidfunk/mkdocs-material#3017] [Q&A]
How can the site be rendered directly from `/docs` with mkdocs-material theme?

`mkdocs gh-deploy --force` creates `user.github.io/repo/site`  but  `user.github.io/repo` renders  `user.github.io/repo/docs/index.md`without mkdocs-material theme (ie vanilla mkdocs).

If source is set to root `user.github.io/repo/site/index.html` renders with mkdocs-material theme.

Using minimal deploy:

```
pip install mkdocs-material
mkdocs new .
mkdocs build
rem from the directory containing the mkdocs.yml file:
mkdocs gh-deploy --force
```

**Comment by squidfunk** [comment]
This question is not specific to Material for MkDocs. You should ask your question on the [MkDocs discussion board](https://github.com/mkdocs/mkdocs/discussions).

**Comment by devzvrm** [comment]
@flywire - any reason you're not changing the site to root on github?

**Comment by flywire** [comment]
Trying to install as described in docs, [possibly didn't wait long enough before changing GitHub Pages Source](https://github.com/mkdocs/mkdocs/discussions/2569#discussioncomment-1350663).

[mkdocs-material docs](https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-mkdocs) leave out a lot of confusing detail but it would be clearer for checking with a _Note: Default install to `gh-pages` branch `root` folder._

**Reply by devzvrm** [reply]
Yeah I'm not sure exactly why the site gets deployed to gh-pages branch as /site vs /docs/site as other templates and the base mkdocs do. Probably could have dug into it, but I saw the discrepancy, updated my repository pages, and it worked out. I'm going to guess it's something in the build process that differs.
